ospec/ospec.js

```diff
@@ -112,12 +110,15 @@ module.exports = new function init() {
 			}
 		}
 	}
-	function unique(subject, predicate) {
-		if (ctx.hasOwnProperty(subject) && predicate) {
-			console.warn("A test named `" + subject + "` was already defined")
-			subject = subject + "*"
+	var dedupeCounter = 0
+	function unique(subject) {
+		var res = subject
+		if (ctx.hasOwnProperty(subject)) {
```

Comment:
Use `Object.prototype.hasOwnProperty` directly here, so it doesn't get screwed up over a test literally named `hasOwnProperty`.